ai_genomics/utils/patent_data/get_ai_genomics_patents_utils.py

```diff
@@ -0,0 +1,84 @@
+"""Script to instantiate Google BigQuery client and
```

Comment:
This looks more like a set of functions than a script (which to me implies that it's run from the command line)